anaUltraScurve.py

```diff
@@ -210,16 +226,61 @@ def overlay_fit(VFAT, CHAN):
     trimrange_list[event.vfatN][event.vfatCH] = event.trimRange
     pass
 
+# Determine hot channels
+from anautilities import isOutlierMADOneSided
+import numpy as np
 if options.SaveFile:
+    print 'Determining hot channels'
+    masksDead = []
+    masksHot = []
     masks = []
+    for vfat in range(0, 24):
+        trimValue = np.zeros(128)
+        dead = np.zeros(128)
+        for ch in range(0, 128):
+            # Get fit results
+            threshold[0] = scanFits[0][vfat][ch]
+            noise[0] = scanFits[1][vfat][ch]
+            pedestal[0] = scanFits[2][vfat][ch]
+            # Identify dead channels
+            dead[ch] = (threshold[0] == 8 and pedestal[0] == 8)
+            # Compute the value to apply MAD on for each channel
+            trimValue[ch] = threshold[0] - options.ztrim * noise[0]
+        masksDead.append(dead)
+        # Determine outliers
+        hot = isOutlierMADOneSided(trimValue, thresh=options.zscore,
+                                   rejectHighTail=False).flatten()
```

Comment:
`isOutlierMADOneSided` returns an array like `[[0], [1], [2], ...]`, and I don't care about the second dimension. I don't think I can change `isOutlierMADOneSided` without breaking `anaUltraThreshold`